science/gr-satellites/Portfile

```diff
@@ -11,16 +11,12 @@ maintainers         {@ra1nb0w irh.it:rainbow} {michaelld @michaelld} openmaintai
 description         GNU Radio decoders for several Amateur satellites
```

Comment:
Would you please add to the description (and a comment with the new livecheck) about that this port is for the 1.X branch? That way we're making it clear that this is not about the 2.Y or 3.Z branches. Thx!

Reply:
is it ok now?
Anyway, my idea as I said through e-mail is to have gr37-satellites for GR37.

Reply:
Yes yes I know ... still good to have the info for the interim ... thx!